pkgs/top-level/all-packages.nix

```diff
@@ -2451,6 +2451,8 @@ with pkgs;
 
   lf = callPackage ../tools/misc/lf {};
 
+  lesscpy = pythonPackages.lesscpy;
```

Comment:
don't add Python libraries here. If its an application, then it should not be called via `python-packages.nix`.